lib/router/index.js

```diff
@@ -7,8 +7,8 @@ module.exports = ({ hooks, middleware }) => {
   const router = Router();
 
   router.use((req, res, next) => {
-    res.respondWith = data => {
-      res.json({ data });
+    res.respondWith = (data, total) => {
+      res.json({ data, total });
     };
```

Comment:
I'd be tempted to make this work like the public api with top level `meta` property that then contains any associated metadata.

Reply:
This would achieve that, but I'm not sure what impact that would have downstream.

```js
res.respondWith = (data, ...meta) => {
  res.json({ data, meta });
};
```